themes/ddbasic/sass/components/ting-object/material-item.scss

```diff
@@ -0,0 +1,242 @@
+@import '../../base.scss';
+
+// ==========================================================================
+// User material item
+// ==========================================================================
+
+
+.material-item {
+  position: relative;
+  float: left;
+  
+  min-height: 136px;
+  width: 100%;
+  padding: 10px 20px;
+  
+  @include clearfix();
+  
+  @include media($mobile){
+    padding: 10px;
+  }
+  
+  .left-column {
+    float: left;
+    
+    min-width: 130px;
+    min-height: 28px;
+    
+    @include media($mobile){
+      position: absolute;
+      top: 0px;
+      left: 0px;
+      min-width: 60px;
+      min-height: 60px;
+    }
+    
+    .item-checkbox {
+      position: absolute;
+      left: 20px;
+      top: 50%;
+      @include transform(translateY(-50%));
+      
+      @include media($mobile){
+        left: 10px;
+        top: 10px;
+        @include transform(translateY(0%));
+      }
+      
+      .form-type-checkbox {
+        position: relative;
+        
+        width: 28px;
+        height: 28px;
+        margin: 0;
+        &.form-disabled {
+          visibility: hidden;
+        }
+        
+        label {
+          text-indent: -9999px;
+          &:after {
+            text-indent: 0;
+          }
+        }
+      }
+    }
+    
+    .ting-cover {
+      float: left;
+      
+      width: 70px;
+      margin-left: 60px;
+      
+      @include media($mobile){
+        display: none;
+      }
+    }
+    
+  }
+  .right-column {
+    float: left;
+    width: calc(100% - 130px);
+    padding-left: 30px;
+    
+    @include media($mobile){
+      width: 100%;
+      padding: 0;
+    }
+    
+    .item-title {
+      width: 100%;
+      float: left;
+      margin: 10px 0 10px 0;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      color: $color-primary;
+      &.has-message {
+        width: calc(100% - 190px);
+        @include media($mobile) {
+          width: 100%;
+        }
+      }
+      #ding-bookmark-remove-form & {
+        width: calc(100% - 170px);
+        @include media($mobile) {
+          width: 100%;
+        }
+      }
+      @include media($mobile) {
+        width: 100%;
+        min-height: 50px;
+        margin-top: 0;
+        padding: 0 10px 0 60px;
+        white-space: normal;
+      }
+    }
+    
+    .messages.warning {
```

Comment:
The styling here is specific for warning messages in material items. A comment has been added.